db/data/20220803164723_fix_end_academic_cycle_id_for_awarded_withdrawn_trainees.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class FixEndAcademicCycleIdForAwardedWithdrawnTrainees < ActiveRecord::Migration[6.1]
+  def up
+    hesa_awarded_trainee_ids = Trainee.imported_from_hesa.awarded.pluck(:id)
```

Comment:
So I decided to exclude the withdrawn HESA trainees because my thinking was we shouldn't really have any affected by this issue. HESA trainees aren't awarded or withdrawn in Reg directly (as far as i'm aware?) so I thought there was no point to include them in the main queries. This is me basically trying to reduce down the scope so we aren't running the SetAcademicCycles job on way too many trainees. I'm not sure I've been super successful on my scoping though.